lib/board.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# Board class
+class Board
+  attr_reader :possible_wins
+  attr_accessor :board
+
+  def initialize
+    @possible_wins = [
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 4, 8],
+      [2, 4, 6]]
```

Comment:
You should put the last closing bracket at the next line, not at the same line than the last element.